.travis.yml

```diff
@@ -4,3 +4,15 @@ go: "1.11"
 install: true
 script:
 - GINKGO_EXTRA_ARGS="--compilers=2" make all test/unit
```

Comment:
Would you please add a scratch building here, while we are at it?

Reply:
the `all` in the make already build a container here